scripts/system_performance/models/query_argument_constructor.py

```diff
@@ -0,0 +1,116 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+"""
+Contains query constructors for system performance check MySQL queries.
+"""
+
+# Dependencies
+from datetime import date, datetime
+from calendar import monthrange
+
+# from scripts.system_performance.data_persistence.reduction_run_queries_test import DatabaseMonitorChecks
+from scripts.system_performance.data_persistence.system_performance_queries import DatabaseMonitorChecks
```

Comment:
Might need to have a look at the way these directories are structured. I think we can simplify this